### DISCUSSION TITLE
Maybe bug?

### DISCUSSION THREAD

**Discussion by PedEchsing** [ViennaRSS/vienna-rss#1731] [General]
Is it no longer possible to right-click in an opened tab and open in preferred browser?
I highlight a post. I r-click, on option to open in firefox. OK, normal. I r-click heading in secondary window of highlighted tab, get option to open in firefox, Again, normal. Now, I open article in new tab. R-click in new tab, only option is reload. NOT normal.

All previous versions allowed opening new tab in prefered browser, from that tab. Is that gone?
I mean if it is, I'll learn to work around it, but it was so convenient to open a new tab in prefered browser for printing, copying, or other browser-based options.

Vienna Version 3.9.0 :7ad9df044: (8175), MacOS Sonoma 14.3

**Comment by PedEchsing** [comment]
OK...operational change, apparently.
Guess I'll just have to get used to r-clicking/option r-clicking the URL bar to open in default browser.
Goodbye convenience.

Gonna close the discussion, now.

**Reply by TAKeanice** [reply]
No need to close this, your point is valid, but give us some time to fix it as we all develop Vienna in our free time.

**Comment by barijaona** [comment]
Fixed now.
